6-async/02-promise-basics/article.md

```diff
@@ -167,7 +174,7 @@ promise.then(
 );
 ```
 
-If we're interested only in successful completions, then we can provide only one argument to `.then`:
+If we're interested only in successful Promises, then we can provide only one function argument to `.then`:
```

Comment:
Why the change `completions` -> `Promises`? Here we're talking about promise *results*.

Reply:
Reverted to "completions".